sdk/vault-client/test/collections.spec.ts

```diff
@@ -1,18 +1,8 @@
 import {expect} from "chai";
 import {AsyncFunc, Context} from "mocha";
-import {Collection} from "../";
-
-
-it('coll', async function () {
```

Comment:
What was that? why did you delete it?

Reply:
It's not a test.
It was a small code I used to generate the baseline for the README but I modified the README manually since then so it's not relevant anymore.